tests/010_uniqueid

```diff
@@ -8,7 +8,7 @@ cat >"$CONFIG" <<EOF
 -v
 
 # Specify a hostname pattern that has a unique id part
---hostname-pattern nerves-%.4s
+--hostname-pattern nerves-%.4s-
```

Comment:
I don't understand this trailing `-` that is now in all these tests. Is this just added for the tests or now a pattern requirement?

Reply:
It's just to check a code path that wasn't getting exercised before. In fact, I don't need to check that code path in every hostname pattern test.

Reply:
I updated test `010_unique_id` to not have the trailing `-` so that it was closer to the previous test. It certainly would be bad if somehow lacking a trailing character ever broke.